cdk/src/open-data-platform/data-plane/schema/schema.sql

```diff
@@ -95,22 +95,44 @@ CREATE INDEX IF NOT EXISTS census_state_geo_id_index ON demographics (state_cens
 CREATE INDEX IF NOT EXISTS census_county_geo_id_index ON demographics (county_census_geo_id);
 CREATE INDEX IF NOT EXISTS geom_index ON demographics USING GIST (geom);
 
+--- Tables related to U.S. demographic information.
+
+-- TODO(breuch): Add insert statement once the zipcode -> demographics
+
+CREATE TABLE IF NOT EXISTS aggregate_us_demographics
```

Comment:
What does each row in this table represent? I'd rather name the table after that (e.g. `parcels` is a table of parcels).

Reply:
With this schema the rows would represent either a zipcode, county, or state (indicated in type column with a string). Each has a geo ID which allows all of them to be unified in a single table. If we would rather organize by the geographic type I could make an individual table for each one, however it doesn't seem necessary here IMO.

Reply:
sounds good!